### PR TITLE
Load policy files from resources

### PR DESCRIPTION
**Pull request by patrickod**
#1549 but with some auto-formatting applied to pass the `development` check.

### CODE REVIEW

File: docs/content/any/project/changelogs/NEXT.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ draft: true
 
 ## `RELEASED_PACKAGE_1` NEW_VERSION
 
+### Java
+
+#### Other bugs & improvements.
+
+A new `loadFilesFromResources` API has been added to allow loading policy source code from resource files contained in your packaged `.jar`. Special thanks to [kovacstamasx](https://github.com/kovacstamasx) for this contribution.
```
Comment (samscott89) [inline docs/content/any/project/changelogs/NEXT.md:17]:
```suggestion
A new `loadFilesFromResources` API has been added to allow loading policy source code from resource files contained in your packaged `.jar`. Special thanks to [`@kovacstamasx`](https://github.com/kovacstamasx) for this contribution.
```

Always best to put the github username as a code literal to skip the spell checker :)

Comment (patrickod) [reply docs/content/any/project/changelogs/NEXT.md:17]: TIL of this trick!

**Comment by DMWhiteley** [comment]
Hello @samscott89 , @patrickod, I work at the same organisation as Tamas.

This PR being outstanding is pinning us to an old, custom version of oso with the fix in place and this is causing us internal issues. 

Is there anything we can do to assist getting this merged asap?

cc: @MFAshby

**Comment by samscott89** [comment]
Hey @DMWhiteley + @kovacstamasx !

Sorry for the delay in getting this done, this fell through the cracks. Will get this in now

**Review by samscott89 — APPROVED** [review]

**Pull request merged**